src/views/Home/props/prizes.js

```diff
@@ -10,19 +10,19 @@ export const topics = [
     title: "Health Hacks",
     icon: <HealthHacksIcon />,
```

Comment:
Yo this is actually super dope! Have a component in as a prop for the JSON object! Didn't catch this in the #97 PR.